AzureCalling/Controllers/JoinCallViewController.swift

```diff
@@ -101,7 +101,7 @@ class JoinCallViewController: UIViewController, UITextFieldDelegate {
             fatalError("Unexpected destination: \(destination)")
         }
 
-        lobbyViewController.callingContext = callingContext
+        lobbyViewController.callingContext = createCallingContextFunction()
```

Comment:
The only question I have to create a new callingContext is if this would ruin the users' `LobbyView` setting when user navigates back from `LobbyView` with inputs `displayName`/`audioSetting`/`speakerSetting` and update the `groupId/team` link in the `JoinView`. When user forward to the `LobbyView` again, the view would lose every previous setting the user input, correct?

Reply:
The behaviour for this use case stays the same as it was before the changes: the preselected audio, video and name will be set to default values and the speakerSetting will be set to the preselected user's value (also, this preselected valule is shared across the app).